default.nix

```diff
@@ -1,15 +1,17 @@
 { pkgs ? import <nixpkgs> {} }:
 with pkgs;
-pkgs.mkYarnPackage {
-  name = "vscode-haskell";
```

Comment:
Ah does this need to match up with the package name inside `package.json`?

Reply:
I don't think it's important actually. (of course line 14 needs to be correct either way)